Cubical/Categories/Displayed/Fibration/Properties.agda

```diff
@@ -1,33 +1,2 @@
 {-# OPTIONS --safe #-}
 module Cubical.Categories.Displayed.Fibration.Properties where
```

Comment:
should this file be deleted?

Reply:
now removed by 27beb225e9c74654820d60dae934796e05dc8ccf